css/index.css

```
@@ -253,3 +254,191 @@ body {
     font-size: 14px;
   }
 }
+
+
+.carousel{
+  position:relative;
+}
+
+.product {
+	max-width: 1343px;
+  height: 613px;
+	margin: auto;
+	padding: 50px 0px 0px 0px;
+	display: grid;
+	grid-template-columns: 58% 42%;
+
```

Quedo rara la tabulación aquí
```suggestion
  max-width: 1343px;
  height: 613px;
  margin: auto;
  padding: 50px 0px 0px 0px;
  display: grid;
  grid-template-columns: 58% 42%;
```